src/metaUtils.cxx

```diff
@@ -36,15 +37,18 @@ inline bool IsBlank(int c)
 #include <fcntl.h>
 
 #if defined(_WIN32) && !defined(__CYGWIN__)
+#define NOMINMAX
```

Comment:
Yes, exactly as stated in the pull request you linked to: This is required because one of the windows headers (something pulled in by winsock2.h) defines a "max" macro. With that in place, the std::numeric_limits<T>::max() (which my code requires) doesn't work anymore. As far as I know, this has no other side effects.